### ISSUE TITLE
How does this actually work?

### ISSUE THREAD

**Issue by Alfuncoot** [onemanbuilds/InstagramBruteforce#1]
Ok so I am fairly new to programming, I just stumbled on this program on Raidforums, my first question is could you provide some installation and usage commands. Also how exactly does this not get IP blocked, I am aware you can add a delay to each attempt, however, I was sure that each block was around 15 minutes or so, so waiting 60 seconds wouldn't make a difference.

**Comment by onemanbuilds** [comment]
I didn't know that this is on raidforums too 👍 (also thanks for the guy who gave me credit).
I tested only this option ([QUESTION] Check only username, firstname, lastname, birthday variations) so there can be problems with the other options. 
There is a v1 and a v2 for this app. 
The version 1 made with the instagram private api made by ping so it checks the accounts slower, but the v2 made with the requests library with a simplier webrequest so it is faster.
If you configure this tool bad or there are some issues you can be blocked but you can enter the amount of seconds you would like to wait between requests.
It checks if the request was right if everything went fine it checks if you are authenticated with the guessed credentials or not.
If the request was bad then it will wait for the given amount of seconds then it will retry the bruteforce with a different credentail via proxy (this isn't the best and fastest method for it but this is just a simple illustration how bruteforce works. there can be lot of errors with other functions etc maybe in the future if i just want to make a fully tested 100% functional instagram bruteforce tool i will make a better one but at the moment i just wanted to show how bruteforce works also this is like an unfinished project from me).
Usage:
You can download the release which contains the exe file on the releases tab otherwise you have to install python. If you downloaded the v2 in the terminal/cmd you have to install the requirements (pip -r install requirements_v2.txt)
Then simply run the python file on windows in cmd in the app's directory (python main_v2.py)
After that you just answer the questions what methods are you want to use etc.

**Comment by Alfuncoot** [comment]
> 
> 
> I didn't know that this is on raidforums too 👍 (also thanks for the guy who gave me credit).
> I tested only this option ([QUESTION] Check only username, firstname, lastname, birthday variations) so there can be problems with the other options.
> There is a v1 and a v2 for this app.
> The version 1 made with the instagram private api made by ping so it checks the accounts slower, but the v2 made with the requests library with a simplier webrequest so it is faster.
> If you configure this tool bad or there are some issues you can be blocked but you can enter the amount of seconds you would like to wait between requests.
> It checks if the request was right if everything went fine it checks if you are authenticated with the guessed credentails or not.
> If the request was bad then it will wait for the given amount of seconds then it will retry the bruteforce with a different credentail via proxy (this isn't the best and fastest method for it but this is just a simple illustration how bruteforce works. there can be lot of errors with other functions etc maybe in the future if i just want to make a fully tested 100% functional instagram bruteforce tool i will make a better one but at the moment i just wanted to show how bruteforce works also this is like an unfinished project from me).
> Usage:
> You can download the release which contains the exe file on the releases tab otherwise you have to install python. If you downloaded the v2 in the terminal/cmd you have to install the requirements (pip -r install requirements_v2.txt)
> Then simply run the python file on windows in cmd in the app's directory (python main_v2.py)
> After that you just answer the questions what methods are you want to use etc.

Ok well thanks for the reply about being blocked out, however, still I might need some installation and usage instructions, honestly if you put some on the main page, I wouldn't have even asked.

Thanks

**Comment by onemanbuilds** [comment]
If you are on windows just download the release and run it, then answer the questions with 1 or 0 then it will start the bruteforce. If you want to run the source code then you have to install python then run the cmd in the app's folder and type pip -r install requirements_v2.txt in the cmd, after that you have to answer the questions then it will start the bruteforce. ( i mainly develop for windows so there can be problems with other operation systems.)

**Comment by Alfuncoot** [comment]
Whenever I try to install it with pip, I get:
ERROR: Could not find a version that satisfies the requirement os (from -r requirements_v2.txt (line 1)) (from versions: none)
ERROR: No matching distribution found for os (from -r requirements_v2.txt (line 1))

**Comment by onemanbuilds** [comment]
i updated the requirements_v2.txt try to download it again and run the same command

**Comment by Alfuncoot** [comment]
Yeah it's still not working, you could make a video show casing how to use this maybe?

**Comment by onemanbuilds** [comment]
If i will have some freetime i will make a guide or a setup.py or something for it.

**Comment by onemanbuilds** [comment]
Now i generated the requirements with pipreqs so now everything should work fine, if still not then download the zip file from the releases tab. This is an unfinished project of mine so i didn't really focus on this tool tbh
